composer.json

```diff
@@ -23,7 +23,9 @@
     "autoload": {
         "psr-4": {
             "RA\\": "src/",
-            "Test\\": "tests/src"
+            "Test\\": "tests/src",
+			"RA\\Models\\": "src/Models",
+            "RA\\Enums\\": "src/Enums"
```

Comment:
RA\Models and RA\Enums are redundant here and can be removed. The PSR-4 autoloader will find the classes by matching the directories and file name to the given namespace.

Reply:
Ah, didn't know this. That's quite nice, I gotta say. I'll make the change.